server/controllers/cashier.js

```diff
@@ -0,0 +1,78 @@
+import '@babel/plugin-transform-regenerator';
+import '@babel/polyfill';
+import pool from '../config/db-config';
+
+class Cashier {
+  static async debitAccount(req, res) {
+    const cashierId = req.userData.id;
+
+    const getCashier = `SELECT * FROM users WHERE id= '${cashierId}';`;
+    const { rows: [cashierGet] } = await pool.query(getCashier);
+   
+
+
+    const transaction = {
+      date: req.body.date,
+      type: 'debit',
+      accountnNumber: req.params.accountNo,
```

Comment:
Take a second look at this line. Did you mean to write "accountnNumber" or "accountNumber"?